VncLib.Wpf/VncLib.Wpf.csproj

```diff
@@ -1,66 +1,54 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{432A5791-7982-498A-B992-B8791AF2AE20}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>VncLib.Wpf</RootNamespace>
-    <AssemblyName>VncLib.Wpf</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFrameworks>net5.0-windows;net48</TargetFrameworks>
+    <AssemblyTitle>VncLib.Wpf</AssemblyTitle>
+    <Product>VncLib.Wpf</Product>
+    <Copyright>
+      original Copyright 2018 The VncLib Authors#
+      modified by Copyright (c) Buerkert Fluid Control Systems 2021
+    </Copyright>
+    <UseWpf>true</UseWpf>
+    <Company>Buerkert</Company>
+    <VersionPrefix>1.1.0</VersionPrefix>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>Buerkert.VncLib</PackageId>
```

Comment:
Is der mit Absicht ohne WPF, weil die ID sich nicht ändern soll? Bei <Product> genauso...

Reply:
ich änder doch jetzt nicht die package id